googlepubsub/src/main/scala/akka/stream/alpakka/googlepubsub/Session.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.googlepubsub
```

Comment:
How about `akka.stream.alpakka.googlecloud.pubsub`, this will match the project name better.